docs/src/examples/components/Avatar/Variations/AvatarExampleName.shorthand.tsx

```diff
@@ -1,6 +1,13 @@
 import React from 'react'
-import { Avatar } from '@stardust-ui/react'
+import { Avatar, StatusIndicator } from '@stardust-ui/react'
 
-const AvatarExampleNameShorthand = () => <Avatar status="Available" name="John Doe" />
+const Available = (
+  <StatusIndicator
+    icon={{ name: 'check', variables: { color: 'white', backgroundColor: 'green' } }}
+    title="Available"
+  />
+)
```

Comment:
This could also be a props object for simplicity, correct?

```jsx
const status = {
  icon: { name: 'check', variables: { color: 'white', backgroundColor: 'green' } },
  title: "Available",
}
```

This removes a few lines and the need to import the status indicator.